src/utility.ts

```diff
@@ -1,12 +1,49 @@
 "use strict";
+import { platform } from "os";
 import * as vscode from "vscode";
 
 export class Utility {
-    public static codeLensEnabled(): boolean {
-        return Utility.getConfiguration().get<boolean>("showCodeLens", true);
+    private static showCodeLens: boolean;
+    private static failed: string;
+    private static passed: string;
+    private static skipped: string;
+
+    public static get codeLensEnabled(): boolean {
+        return Utility.showCodeLens;
+    }
+
+    public static get codeLensFailed(): string {
+        return Utility.failed;
+    }
+
+    public static get codeLensPassed(): string {
+        return Utility.passed;
+    }
+
+    public static get codeLensSkipped(): string {
+        return Utility.skipped;
     }
 
     public static getConfiguration(): vscode.WorkspaceConfiguration {
         return vscode.workspace.getConfiguration("dotnet-test-explorer");
     }
+
+    public static updateCache() {
+        const configuration = Utility.getConfiguration();
+        const osx = platform() === "darwin";
+
+        Utility.showCodeLens = configuration.get<boolean>("showCodeLens", true);
+        Utility.failed = Utility.getLensText(configuration, "codeLensPassed", "\u274c"); // Cross Mark

```

Comment:
codeLensPassed?

Reply:
Good catch, I've actually tested them all this time before submitting (and had my ☕️!)